collection/rancher/v2.x/profile-collector/continuous_profiling.sh

```diff
@@ -44,22 +46,27 @@ help() {
 
   All flags are optional
 
-  -a    Application, either rancher or cattle-cluster-agent
+  -a    Application: rancher, cattle-cluster-agent, fleet-controller, or fleet-agent
   -p    Profiles to be collected (comma separated): goroutine,heap,threadcreate,block,mutex,profile
   -s    Sleep time between loops in seconds
   -t    Time of CPU profile collections
   -h    This help"
 
 }
 
+timestamp() {
```

Comment:
Seems like this function is called in `techo` exclusively. Unless you need it somewhere else, I suggest to inline it

Reply:
inlined timeline on techo